packages/reporters/dev-server/src/Server.js

```diff
@@ -495,6 +495,17 @@ export default class Server {
       setHeaders(res);
       next();
     });
+
+    app.use((req, res, next) => {
+      if (req.url === '/healthcheck') {
```

Comment:
Could conflict with a user's URL (eg static page or proxy), no? Maybe we should prefix with __parcel_ like we do for the HMR endpoint?

Reply:
Let me do that :+1: